include/uapi/sound/sof/header.h

```diff
@@ -13,8 +13,12 @@
 
 /**
  * struct sof_abi_hdr - Header for all non IPC ABI data.
- * @magic: Magic number for validation: 0x00464F53 ('S', 'O', 'F', '\0')
- * @type: Component specific type
+ * @magic: Magic number for validation
+ *	   for IPC3 data: 0x00464F53 ('S', 'O', 'F', '\0')
+ *	   for IPC4 data: 0x34464F53 ('S', 'O', 'F', '4')
+ * @type: module specific parameter
+ *	  for IPC3: Component specific type
+ *	  for IPC4: parameter ID (param_id) of the data
  * @size: The size in bytes of the data, excluding this struct
  * @abi: SOF ABI version
```

Comment:
nit-pick: we should clarify that the ABI version is within the scope of the magic number, i.e. there is no relationship between the abi value for IPC3 and for IPC4.

Reply:
Yes, it is a good idea, I will create a fixup for this.